migrations/2018-08-28-010336_init/up.sql

```diff
@@ -26,7 +26,7 @@ CREATE TABLE `bso` (
 CREATE TABLE `collections` (
     `id` INT PRIMARY KEY      NOT NULL AUTO_INCREMENT,
     `name` VARCHAR(32) UNIQUE NOT NULL
-) ENGINE=InnoDB AUTO_INCREMENT=100 DEFAULT CHARSET=latin1;
```

Comment:
Isn't it bad form to update an old migration?

Reply:
Definitely but with AUTO_INCREMENT being ephemeral it doesn't really matter -- which is an argument for leaving it -- but I'm nuking it so no one might mistake it for being needed/useful